packages/cli/src/commands/setup/deploy/providers/flightcontrol.js

```diff
@@ -219,12 +224,13 @@ const updateApp = () => {
       )
       if (authLineIndex === -1) {
         console.log(`
-    Couldn't find <AuthProvider in web/src/App.js
-    You'll have to add the following fetch config manually:
+    Couldn't find <AuthProvider /> in web/src/App.js
+    If (and when) you use *dbAuth*, you'll have to add the following fetch config to <AuthProvider />:
 
     config={{ fetchConfig: { credentials: 'include' } }}
     `)
-      } else {
+        // This is CORS config for cookies, which is currently only dbAuth Currently only dbAuth uses cookies and would require this config
+      } else if (appContent.toString().match(/dbAuth/)) {
```

Comment:
@flybayer I missed this the first time around — glad I caught it now! 🤦‍♂️

These changes to App.js|tsx are only required *if* dbAuth is being used because dbAuth is cookie auth. So I updated the logic here to check for dbAuth in the provider. Same for L251

Please check my work and my copy. Can be updated but definitely wanted to make sure the logic was fixed before 1.0